frontend/src/pages/project/create.vue

```diff
@@ -14,6 +14,8 @@
                     </template>
                 </FormRow>
 
+                <FormRow :options="stacks" v-model="input.stack" id="stack">Stack</FormRow>
+
```

Comment:
Not sure where to put this, but if you try to create a project with no stacks defined it just silently fails (400 error in the browser console)